source/configure/plugins-configuration-settings.rst

```diff
@@ -1619,8 +1423,8 @@ In addition to the Mattermost plugin signing key built into the server, each pub
 | This feature's ``config.json`` setting is ``"SignaturePublicKeyFiles": {}`` with string array input consisting of contents that are relative or absolute paths to signature files.              |
 +-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
-.. config:setting:: plugins-zoomchimeraoauthproxyurl
-  :displayname: Chimera OAuth proxy URL (Plugins - Zoom)
+.. config:setting:: plugins-chimeraoauthproxyurl
```

Comment:
It is unclear to me which plugin these belong to and when an admin should modify these values. Can we clarify that?

Reply:
The code highlighted is metadata used to return config settings in search results. 

I agree that the [Chimera OAuth proxy URL](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/7030/configure/plugins-configuration-settings.html#chimera-oauth-proxy-url) docs don't specify which integrations this config affects, and we can definitely add this. Can you confirm which integrations utilize this config setting, please?